### PR TITLE
TC14 - User can book many tickets at a time

### PR DESCRIPTION
**Pull request by cvan8397**

### CODE REVIEW

File: src/main/java/page_objects/BookTicketPage.java [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
     private final Label lblBookTicket = new Label(By.cssSelector("#content>h1"));
     private final Label lblBookTicketForm = new Label(By.cssSelector("fieldset>legend"));
+    private final DropDown ddDepartDate = new DropDown(By.name("Date"));
+    private final DropDown ddDepartFrom = new DropDown(By.name("DepartStation"));
+    private final DropDown ddArriveAt = new DropDown(By.name("ArriveStation"));
+    private final DropDown ddSeatType = new DropDown(By.name("SeatType"));
+    private final DropDown ddTicketAmount = new DropDown(By.name("TicketAmount"));
+    private final Button btnBookTicket = new Button(By.cssSelector("fieldset input"));
+    private final Label lblSuccessfulMessage = new Label(By.cssSelector("#content>h1"));
+    private final String dynamicTableCell = "//table/tbody/tr/td[count(//table/tbody/tr/th[.='%s']/preceding-sibling::th)+1]";
```
Comment (tanmle) [inline src/main/java/page_objects/BookTicketPage.java]: not a good XPATH, add more property

File: src/test/java/tests/TC14_UserCanBookManyTicketAtATime.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import page_objects.LoginPage;
+import page_objects.RegisterPage;
+
+public class TC14_UserCanBookManyTicketAtATime extends TestBase {
+    HomePage homePage = new HomePage();
+    RegisterPage registerPage = new RegisterPage();
+    LoginPage loginPage = new LoginPage();
+    BookTicketPage bookTicketPage = new BookTicketPage();
+
+    String departDate = DataHelper.getRandomDepartDate();
```
Comment (tanmle) [inline src/test/java/tests/TC14_UserCanBookManyTicketAtATime.java]: we dont need to define `getRandomDepartDate` in DataHelper.

Comment (cvan8397) [reply src/test/java/tests/TC14_UserCanBookManyTicketAtATime.java]: I have fixed it

File: src/main/java/page_objects/BookTicketPage.java (hunk 1 above)
Comment (cvan8397) [reply src/main/java/page_objects/BookTicketPage.java]: I have fixed my xpath

**Review by tanmle — DISMISSED** [review]

File: src/main/java/page_objects/BookTicketPage.java [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
     private final Label lblBookTicket = new Label(By.cssSelector("#content>h1"));
     private final Label lblBookTicketForm = new Label(By.cssSelector("fieldset>legend"));
+    private final DropDown ddDepartDate = new DropDown(By.name("Date"));
+    private final DropDown ddDepartFrom = new DropDown(By.name("DepartStation"));
+    private final DropDown ddArriveAt = new DropDown(By.name("ArriveStation"));
+    private final DropDown ddSeatType = new DropDown(By.name("SeatType"));
+    private final DropDown ddTicketAmount = new DropDown(By.name("TicketAmount"));
+    private final Button btnBookTicket = new Button(By.cssSelector("fieldset input"));
+    private final Label lblSuccessfulMessage = new Label(By.cssSelector("#content>h1"));
+    private final String dynamicTableCell = "//td[count(//th[text()='%s']/preceding-sibling::th)+1]";
```
Comment (tanmle) [inline src/main/java/page_objects/BookTicketPage.java]: still not a good one.

File: src/main/java/page_objects/BookTicketPage.java (hunk 1 above)
Comment (tanmle) [reply src/main/java/page_objects/BookTicketPage.java]: Still not a good one.

File: src/main/java/page_objects/BookTicketPage.java (hunk 3 above)
Comment (cvan8397) [reply src/main/java/page_objects/BookTicketPage.java]: I have fixed my xpath

**Pull request merged**